packages/material-ui-system/README.md

```diff
@@ -1,17 +1,17 @@
 # @material-ui/system
 
-Style props functions for building powerful design systems.
+CSS utilities for rapidly laying out custom designs.
 
 ## Installation
 
 Install the package in your project directory with:
 
```

Comment:
I thought it works only on the next.js pages, thanks.

Reply:
I kind of wonder how we could improve communication in that regard. Maybe like a little internal ticker where we announce contributing related changes. I know that github adds a notification when CONTRIBUTING.md changes but that doesn't list the actual changes. I wouldn't create a dedicated document since that'll just grow indefinitely and collects outdated stuff. 

Because as a collaborator you're not necessarily following each production related change. But dev changes might be more interesting.

Reply:
Hard problem, I guess it's one of the values of doing code reviews